test/k8sT/Services.go

```diff
@@ -1110,6 +1110,12 @@ var _ = Describe("K8sServicesTest", func() {
 		})
 
 		SkipItIf(helpers.RunsWithoutKubeProxy, "Tests NodePort (kube-proxy) with externalTrafficPolicy=Local", func() {
+			DeployCiliumOptionsAndDNS(kubectl, ciliumFilename, map[string]string{
+				// When kube-proxy is enabled, the host firewall is not
+				// compatible with externalTrafficPolicy=Local because traffic
+				// from pods to remote nodes goes through the tunnel.
+				"global.hostFirewall": "false",
+			})
```

Comment:
Are those incompatibilities (this one, and with portmap chaining) documented somewhere?

Reply:
Not yet. I have several reminders to create specific issues for those incompatibilities once the present PR is merged. I would prefer to avoid creating those issues before in case another approach is found through the reviews.